Should have an M1 compatible build or a Universal Binary

It would be great to have an M1 compatible build or a Universal Binary.

@dorkrawk How were you able to build Crystalline on an M1? I get these errors when I try:

```
Undefined symbols for architecture arm64:
  "llvm::DISubprogram::toSPFlags(bool, bool, bool, unsigned int, bool)", referenced from:
      _LLVMExtDIBuilderCreateFunction in llvm_ext.o
  "llvm::DIBuilder::createFunction(llvm::DIScope*, llvm::StringRef, llvm::StringRef, llvm::DIFile*, unsigned int, llvm::DISubroutineType*, unsigned int, llvm::DINode::DIFlags, llvm::DISubprogram::DISPFlags, llvm::MDTupleTypedArrayWrapper<llvm::DITemplateParameter>, llvm::DISubprogram*, llvm::MDTupleTypedArrayWrapper<llvm::DIType>, llvm::MDTupleTypedArrayWrapper<llvm::DINode>, llvm::StringRef)", referenced from:
      _LLVMExtDIBuilderCreateFunction in llvm_ext.o
  "llvm::DIBuilder::createMemberType(llvm::DIScope*, llvm::StringRef, llvm::DIFile*, unsigned int, unsigned long long, unsigned int, unsigned long long, llvm::DINode::DIFlags, llvm::DIType*, llvm::MDTupleTypedArrayWrapper<llvm::DINode>)", referenced from:
      _LLVMExtDIBuilderCreateMemberType in llvm_ext.o
  "llvm::DIBuilder::createPointerType(llvm::DIType*, unsigned long long, unsigned int, llvm::Optional<unsigned int>, llvm::StringRef, llvm::MDTupleTypedArrayWrapper<llvm::DINode>)", referenced from:
      _LLVMExtDIBuilderCreatePointerType in llvm_ext.o
  "llvm::DIBuilder::createParameterVariable(llvm::DIScope*, llvm::StringRef, unsigned int, llvm::DIFile*, unsigned int, llvm::DIType*, bool, llvm::DINode::DIFlags, llvm::MDTupleTypedArrayWrapper<llvm::DINode>)", referenced from:
      _LLVMExtDIBuilderCreateParameterVariable in llvm_ext.o
  "llvm::DIBuilder::createReplaceableCompositeType(unsigned int, llvm::StringRef, llvm::DIScope*, llvm::DIFile*, unsigned int, unsigned int, unsigned long long, unsigned int, llvm::DINode::DIFlags, llvm::StringRef, llvm::MDTupleTypedArrayWrapper<llvm::DINode>)", referenced from:
      _LLVMExtDIBuilderCreateReplaceableCompositeType in llvm_ext.o
  "llvm::AttributeList::addAttributeAtIndex(llvm::LLVMContext&, unsigned int, llvm::Attribute::AttrKind) const", referenced from:
      llvm::IRBuilderBase::CreateCall(llvm::FunctionType*, llvm::Value*, llvm::ArrayRef<llvm::Value*>, llvm::ArrayRef<llvm::OperandBundleDefT<llvm::Value*>>, llvm::Twine const&, llvm::MDNode*) in llvm_ext.o
      llvm::IRBuilderBase::CreateInvoke(llvm::FunctionType*, llvm::Value*, llvm::BasicBlock*, llvm::BasicBlock*, llvm::ArrayRef<llvm::Value*>, llvm::ArrayRef<llvm::OperandBundleDefT<llvm::Value*>>, llvm::Twine const&) in llvm_ext.o
  "llvm::AttributeList::addAttributeAtIndex(llvm::LLVMContext&, unsigned int, llvm::StringRef, llvm::StringRef) const", referenced from:
      _LLVMExtCreateMCJITCompilerForModule in llvm_ext.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Error: execution of command failed with exit status 1: cc "${@}" -o /Users/avrameisner/Dev/crystal-code/crystalline/bin/crystalline  -rdynamic -L/opt/homebrew/Cellar/crystal/1.8.2/libexec/../../../../lib -L/opt/homebrew/Cellar/libyaml/0.2.5/lib -lyaml /opt/homebrew/Cellar/crystal/1.8.2/share/crystal/src/llvm/ext/llvm_ext.o `"/opt/homebrew/opt/llvm@13/bin/llvm-config" --libs --system-libs --ldflags 2> /dev/null` -lstdc++ -lpcre2-8 -lgc -L/opt/homebrew/Cellar/libevent/2.1.12/lib -levent_pthreads -levent -L/opt/homebrew/Cellar/libevent/2.1.12/lib -levent -liconv
```

--- a/README.md
+++ b/README.md
@@ -62,12 +62,12 @@ chmod u+x crystalline
 yay -S crystalline
 ```
 
-##### MacOS (x86_64)
+##### MacOS
+
+Install using [homebrew](https://brew.sh):
 
 ```sh
-curl -L https://github.com/elbywan/crystalline/releases/latest/download/crystalline_x86_64-apple-darwin.gz -o crystalline.gz &&\
-gzip -d crystalline.gz &&\
-chmod u+x crystalline
+brew install crystalline
 ```
 
 #### Specific release

